src/function_manager.jl

```diff
@@ -77,14 +77,15 @@ end
 
 function export_function!(fm::FunctionManager, f, job_id=get_job_id())
     fd = ray_jll.function_descriptor(f)
+    function_locations = functionloc.(methods(f))
     key = function_key(fd, job_id)
-    @debug "Exporting function to function store:" fd key
+    @debug "Exporting function to function store:" fd key function_locations
     # DFK: I _think_ the string memory may be mangled if we don't `deepcopy`. Not sure but
     # it can't hurt
     if ray_jll.Exists(fm.gcs_client, FUNCTION_MANAGER_NAMESPACE, deepcopy(key), -1)
-        @debug "Function already present in GCS store:" fd key f
+        @debug "Function already present in GCS store:" fd key
     else
-        @debug "Exporting function to GCS store:" fd key f
+        @debug "Exporting function to GCS store:" fd key
```

Comment:
Avoid including the function in the log messages as these can get extremely verbose: https://github.com/beacon-biosignals/Ray.jl/actions/runs/6484894901/job/17609819142#step:12:274